pyproject.toml

```diff
@@ -46,13 +46,20 @@ mesh = [
     "pyvista",
     "meshio",
 ]
+lddmm = [
+    "torch",
+    "matplotlib",
+    "pyvista",
+    "deformetrica @ git+https://github.com/nguigs/deformetrica@spline-regression",
+    "PyQt5"
+]
 varifold = ["geomstats", "kneebow", "trimesh"]
 dataframe = ["pandas"]
 ssh = ["paramiko", "scp"]
 preprocessing = ["polpo[mri,mesh,dataframe,ssh,point_cloud]"]
 hydra = ["hydra-core", "omegaconf"]
 dev = ["polpo[doc,lint,test]"]
-all = ["polpo[dash,app,plot,preprocessing,hydra,varifold]"]
```

Comment:
adding lddmm here will mess up the installation: it will downgrade most of the packages to very old versions

Reply:
can't understand the problem with `backports`!